src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon(){
+        return (isDone ? "X" : " ");
```

Comment:
Excellent use of an in-line conditional expression to make the code more intuitive!